### ISSUE TITLE
BCQ vs Discrete BCQ

### ISSUE THREAD

**Issue by stijnmeels** [takuseno/d3rlpy#374]
Dear @takuseno,

When I am training my model with BCQ (actions normalized in a [0,1] range) I get pretty decent results but not exactly what I hope to achieve.
However when I do not normalize these actions as they are discrete numbers (a price to set) between [0,1000] and I am training a discrete BCQ model the reported loss and imitator_loss are almost constant even though learning rate is increased a bit (various attempts up to 0.01 instead of 0.0000625) from the standard value. When letting the trained agent take actions on a validation set it also returns only the maximum price of 1000$.
What are possible reasons that without making any differences the model is able to learn in continuous action space but not in discrete action space.  Note that I want to compare the difference between a discrete and continuous action space in terms of performance.

Kind regards,

Stijn

**Comment by takuseno** [comment]
@stijnmeels Hi, thanks for the issue and sorry for the late response. My guess is that your action dimension is too large (1000 dims). Basically, it's not easy to work with this size of discrete actions. I'd suggest you to stick to the continuous action-space.

**Comment by takuseno** [comment]
Since this is not an issue of d3rlpy, but more like a general RL question, let me close this issue.
